verifier/trustpolicy/trustpolicy.go

```diff
@@ -265,7 +266,7 @@ func (trustPolicyDoc *Document) GetApplicableTrustPolicy(artifactReference strin
 func LoadDocument() (*Document, error) {
 	jsonFile, err := dir.ConfigFS().Open(dir.PathTrustPolicy)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Trust policy is not present, please create trust policy at %s", filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy))
```

Comment:
If the trustpolicy.json file has a permission issue, what error should be returned? I think we need to keep the original error.

Reply:
Good point, we should should check for the file permission and return better error message.
The original error message `Error: open /Users/pritesb/Library/Application Support/notation/trustpolicy.json: no such file or directory` is not actionable.